app/globals.css

```diff
@@ -137,3 +137,30 @@
 .footer__copyrights-link {
   @apply flex-1 flex sm:justify-end justify-center max-sm:mt-4 gap-10;
 }
+/* Recursion css styles */
+.petal-rotating {
```

Comment:
Can you move this in a CSS file next to the tested component `recursion.css` and then import it in `Recursion.tsx` by doing `import "./recursion.css"?

Reply:
Moved, thank you